Gopkg.toml

```diff
@@ -55,12 +55,46 @@ required = [
   name = "github.com/hashicorp/vault"
   version = "v0.9.3"
 
+[[constraint]]
+  name = "sigs.k8s.io/controller-runtime"
+  # temporarily, before release
+  branch = "master"
```

Comment:
Can we pin to a specific revision here? Makes updates in future at least more predictable 😄

Reply:
yep, that's the plan, was just doing this during development and forgot to switch it back.